ReleaseTooling/Sources/ManifestParser/main.swift

```diff
@@ -69,9 +66,7 @@ struct ManifestParser: ParsableCommand {
       try parsePodNames(FirebaseManifest.shared)
     case .forGHAMatrixGeneration:
       guard let sdkRepoURL = SDKRepoURL else {
-        throw fatalError(
```

Comment:
Removing `throw` fixes the warning `Will never be executed: A call to a never-returning function`.